lldb/tools/lldb-dap/Protocol/ProtocolBase.cpp

```diff
@@ -1,12 +1,12 @@
-//===-- Protocol.cpp ------------------------------------------------------===//
+//===-- ProtocolBase.cpp --------------------------------------------------===//
```

Comment:
Why not just Protocol? Do you think that's to confusing with the other Protocol classes?

Reply:
I used the names of the categories from the spec for the files, https://microsoft.github.io/debug-adapter-protocol/specification if you look at the navigation on the left its grouped into 'Base Protocol', 'Events', 'Requests', 'Reverse Requests' and 'Types'.

Reply:
Oh right, I didn't make the connection. Sounds good.